File: uri 1074.py
N = int(input())
j = 1
while j <= N:
    num = int(input())
    if num == 0:
        print("NULL")
    elif num % 2 == 0 > num:
        print("EVEN NEGATIVE")
    elif num % 2 != 0 > num:
        print("ODD NEGATIVE")
    elif num % 2 == 0 < num:
        print("EVEN POSITIVE")
    elif num % 2 != 0 < num:
        print("ODD POSITIVE")
    j = j + 1

File: uri 1074.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int N,i,b;
    scanf("%d",&N);
    for(i=0;i<N;i++){
        scanf("%d",&b);
        if(b==0)
            printf("NULL\n");
        else{
            if( (b%2)==0 && b>0)
                printf("EVEN POSITIVE\n");
            if( (b%2)==0 && b<0)
                printf("EVEN NEGATIVE\n");
            if( (b%2)!=0 && b<0)
                printf("ODD NEGATIVE\n");
            if( (b%2)!=0 && b>0)
                printf("ODD POSITIVE\n");
        }
    }
    return 0;
}
